lib/Servant/JuicyPixels.hs

```diff
@@ -61,6 +64,26 @@ instance (KnownNat quality, quality <= 100, ColorSpaceConvertible a PixelYCbCr8)
 instance (KnownNat quality, quality <= 100) => MimeUnrender (JPEG quality) DynamicImage where
     mimeUnrender _ = decodeJpeg . BL.toStrict
 
+data JPEG_METADATA (quality :: Nat)
```

Comment:
Why do you introduce a new type here? Shouldn't it be possible to add the instances below to the existing one?

Reply:
And would you mind adding this for the other ContentTypes as well?